hatchet_sdk/loader.py

```diff
@@ -184,6 +186,17 @@ def get_config_value(key, env_var):
             == "True"
         )
 
+
+        #  Add preset labels to the worker config
+        worker_preset_labels: dict[str, str] = defaults.worker_preset_labels
+
+        autoscaling_target = get_config_value(
+            "autoscaling_target", "HATCHET_CLIENT_AUTOSCALING_TARGET"
```

Comment:
Do we want this env var to be `HATCHET_CLIENT_AUTOSCALING_TARGET` or `HATCHET_AUTOSCALING_TARGET`